tests/integration/targets/mount/tasks/main.yml

```diff
@@ -406,3 +420,283 @@
     - /tmp/myfs1
     - /tmp/test_fstab
   when: ansible_system in ('Linux')
+
+######################################################################################################
+######################################################################################################
+######################################################################################################
```

Comment:
It would be better to remove the three-line separator (#). Because test logic for the ephemeral option has already been separated by the subsequent block :)

Reply:
This will be removed in the next push (squashed). This was just eye-candy during development, sorry for that!